templates/my.cnf.epp

```diff
@@ -0,0 +1,22 @@
+<%- |
+  String $cnf_user,
+  Variant[Sensitive[String],String] $cnf_password,
+  Stdlib::Port $cnf_port,
+  Stdlib::Host $cnf_host,
+  Optional[Stdlib::Absolutepath] $cnf_socket = undef,
+| -%>
```

Comment:
can you add a comment like `# THIS FILE IS MANAGED BY PUPPET`?

Reply:
sure

Reply:
Done.